### PR TITLE
update api documentation for raw event

### PR DESCRIPTION
**Pull request by maurycy**

### CODE REVIEW

File: docs/reference/terminal49/terminal49.v1.json [hunk 1]
```diff
@@ -6236,25 +6260,34 @@
...
                 "type": "string",
                 "format": "date",
-                "description": "The estimated date of the event at the event location when no time information is available. "
+                "description": "Deprecated: The estimated date of the event at the event location when no time information is available. "
               },
               "actual_at": {
                 "type": "string",
                 "format": "date-time",
-                "description": "The datetime the event transpired in UTC"
+                "description": "Depracated: The datetime the event transpired in UTC"
```
Comment (mattyturner) [inline docs/reference/terminal49/terminal49.v1.json:6285]: `Deprecated`

File: docs/reference/terminal49/terminal49.v1.json [hunk 2]
```diff
@@ -6236,25 +6260,34 @@
...
               },
               "estimated_at": {
                 "type": "string",
                 "format": "date-time",
-                "description": "The estimated datetime the event will occur in UTC"
+                "description": "Depracated: The estimated datetime the event will occur in UTC"
```
Comment (mattyturner) [inline docs/reference/terminal49/terminal49.v1.json:6290]: `Deprecated`

**Review by mattyturner — CHANGES_REQUESTED** [review]
Typo :)

File: docs/reference/terminal49/terminal49.v1.json (hunk 2 above)
Comment (maurycy) [reply docs/reference/terminal49/terminal49.v1.json]: @mattyturner 🤦🏼 Fixed!

File: docs/reference/terminal49/terminal49.v1.json (hunk 1 above)
Comment (maurycy) [reply docs/reference/terminal49/terminal49.v1.json]: @mattyturner 🤦🏼 Fixed!

**Pull request merged**